src/app/page.js

```diff
@@ -60,5 +63,9 @@ export default function Home() {
         </a>
       </div>
     </main>
+      <div className="flex min-h-screen min-w-screen flex-col items-center  xl:p-0 md:text-2xl bg-black shadow"  >
```

Comment:
Needs a little more padding from top and takes up height more than it needs
![image](https://github.com/Hackodisha/Hackodisha2023/assets/75237697/45d8f5fe-cc3a-46c2-b932-29ffa0c28b62)